test.properties

```diff
@@ -7780,22 +7890,13 @@
         Source Formatter
 
     testray.team.frontend-infrastructure.component.names=\
-        A11y Tool,\
-        AMD Loader,\
-        AUI,\
-        Client Extension Framework,\
         Client Extensions > Custom Element,\
-        Frontend Custom Dialog,\
+        Frontend Client Extensions,\
```

Comment:
@izaera IMHO, It should be better if those names match with Jira components, in that way it is going to be easier for other Teams (Release Teams) assign the proper component when they are analyzing errors. https://liferay.atlassian.net/jira/software/c/projects/LPD/components?filter=&orderDirection=DESC&orderField=NAME&page=1

Reply:
Yeah, makes sense. I will try to make someone rename JIRA components, then :crossed_fingers: .